### PR TITLE
Simplify RPATH

### PR DESCRIPTION
**Pull request by guparan**

### CODE REVIEW

File: CMake/SofaPython3Tools.cmake [hunk 1]
```diff
@@ -201,34 +202,39 @@ function(SP3_add_python_module)
...
-            if (NOT "${dependency_path_from_packages}" STREQUAL "" AND NOT "${dependency_path_from_packages}" STREQUAL "../")
-                list(APPEND ${A_TARGET}_DEPENDENCIES_RPATH "$ORIGIN/../${dependency_path_from_packages}")
-                list(APPEND ${A_TARGET}_DEPENDENCIES_RPATH "$$ORIGIN/../${dependency_path_from_packages}")
-                list(APPEND ${A_TARGET}_DEPENDENCIES_RPATH "@loader_path/../${dependency_path_from_packages}")
-                list(APPEND ${A_TARGET}_DEPENDENCIES_RPATH "@executable_path/../${dependency_path_from_packages}")
-            endif()
-        endif()
-    endforeach()
+    get_target_property(TARGET_DEPENDENCIES_RPATH ${A_TARGET} "INSTALL_RPATH")
+
```
Comment (jnbrunet) [inline CMake/SofaPython3Tools.cmake:206]: You removed the loop on the SP3 dependencies? Won't this break when, for example, 
```
lib/python3/site-packages/SofaTypes/SofaTypes.so
```
 depends on 
```
lib/python3/site-packages/Sofa/Core.so
```

Comment (guparan) [reply CMake/SofaPython3Tools.cmake:206]: Ok that was the purpose of this loop. Thanks, I will put it back :laughing:

Comment (guparan) [reply CMake/SofaPython3Tools.cmake:206]: Done !

**Review by jnbrunet — APPROVED** [review]
Looking good, ready when you are @guparan (let's also the CI finish first)

**Pull request merged**